ansible/roles/nim-waku/templates/docker-compose.yml.j2

```diff
@@ -41,6 +41,9 @@ services:
       --persist-messages=true
       --lightpush=true
       --swap={{ nim_waku_swap_protocol_enabled | lower }}
+      --websocket-secure-support=true
+      --websocket-secure-key-path={{ nim_waku_websockify_ssl_key }}
+      --websocket-secure-cert-path={{ nim_waku_websockify_ssl_cert }}
```

Comment:
These files do not exist within the container in the current configuration. Look at the `websockify` and how it's set up.

Reply:
Also, this is a new flag so it needs to not be enabled for `prod`, since there it would fail.

Reply:
> These files do not exist within the container in the current configuration. Look at the `websockify` and how it's set up.

I added the volumes section for ssl_directory.

Reply:
> Also, this is a new flag so it needs to not be enabled for `prod`, since there it would fail.

How do you recommend we proceed with the same.

Reply:
We need flags for this config values. For enabling it, and for setting port and address.
Ones like `nim_waku_websocket_enabled`. Sames goes for `nim_waku_websocket_ssl_key` which can be for now set to `nim_waku_websockify_ssl_key`, and so on.

For per-fleet configuration, this can be done vie groups files:
https://github.com/status-im/infra-nim-waku/blob/master/ansible/group_vars/wakuv2-test.yml
https://github.com/status-im/infra-nim-waku/blob/master/ansible/group_vars/wakuv2-prod.yml

Reply:
added in 1de14c25e80c478f2163bf5e8deacccee8a6e81d